lib/board.rb

```diff
@@ -1,3 +1,5 @@
+require './lib/cell'
+
```

Comment:
I thought that class files that lived in the same folder didn't need to be required like this. I'll still go ahead ab merge these changes.